examples/undo/example_editor_manager.py

```diff
@@ -42,16 +42,16 @@ def create_control(self, parent):
         self._set_text(w)
         self._set_size_and_style(w)
 
-        self.obj.on_trait_change(self._update_text, 'text')
-        self.obj.on_trait_change(self._update_size, 'size')
-        self.obj.on_trait_change(self._update_style, 'style')
+        self.obj.observe(self._update_text, 'text')
+        self.obj.observe(self._update_size, 'size')
+        self.obj.observe(self._update_style, 'style')
 
         return w
 
     def _name_default(self):
         return self.obj.text
 
-    def _update_text(self):
+    def _update_text(self, _=None):
```

Comment:
Unless `_update_text` is called directly without arguments, let's go with this - 

```suggestion
    def _update_text(self, event):
```